src/app/shared/components/category-navigation/category-navigation.component.scss

```diff
@@ -13,14 +13,58 @@
     }
 
     .categories {
+      text-align: center;
       display: grid;
       gap: 1rem;
       grid-template-columns: repeat(auto-fit, minmax(120px, 121px));
       justify-content: space-between;
     }
+
+    .categories-mobile {
+      text-align: left;
+
+      .selected-category {
+        position: relative;
+        top: -6px;
+        left: 15px;
+        color: $c-darkblue;
+        font-weight: bold;
+      }
+
+      mat-form-field {
+        width: 100%;
+
+        ::ng-deep {
```

Comment:
`::ng-deep` is deprecated, is there a way we can avoid it? So far, we have only one usage in the project.

Reply:
@pkwasnik  natywne style materialowe nadpisujemy raczej w styles/overrides - nie trzeba wtedy dawać ng-deep